apps/src/lab2/views/components/Instructions.tsx

```diff
@@ -47,11 +49,15 @@ const Instructions: React.FunctionComponent<InstructionsProps> = props => {
   );
   const levelIndex = useSelector(currentLevelIndex);
   const currentLevelCount = useSelector(levelCount);
-  const validationState = useSelector(
+  const {hasConditions, message, satisfied} = useSelector(
     (state: {lab: LabState}) => state.lab.validationState
   );
+
+  // If there are no validation conditions, we can show the next button so long as
+  // there is another level. If validation is present, also check that conditions are satisfied.
   const showNextButton =
-    validationState.satisfied && levelIndex + 1 < currentLevelCount;
+    (!hasConditions || satisfied) && levelIndex + 1 < currentLevelCount;
```

Comment:
@molly-moen @breville want to get a quick gut check from y'all on this - once again trying to navigate that divide between keeping things simple for labs vs being too prescriptive and locking ourselves into patterns that we may have to undo later. We're making an assumption here that if a level has no validation conditions, then we are ok to show the continue button (so long as there is another level). On one hand this keeps things simple as neither AI Chat nor Music Lab has to worry about specifying the conditions under which to show the button, but this does introduce more conditional logic here.

Reply:
This seems reasonable, especially for now.

One nuance we might face later is that we probably want Music Lab's "Continue" button to appear a little bit later than when we hit the success condition, so that the user can see/hear the successful condition and then get the reinforcement of that button appearing.  It feels that similarly there might be some benefit in making the "Continue" button appear with a slight delay in these AI Chat levels, even if technically the user can skip immediately.  But maybe we should leave such delays up to the lab implementations.

Reply:
Yeah, that makes sense. I was thinking about if that delay technically qualifies as a validation condition ('user has to wait x seconds') and handling it through the progress system, or if each lab should just handle it in their own way. I'm sure we'll revisit this as we encounter more use cases.

Reply:
I think so long as the lab can provide the validation function and set `satisfied` as it chooses, that still gives us enough flexibility per lab.